src/gui/search/searchjobwidget.h

```diff
@@ -89,6 +89,12 @@ class SearchJobWidget final : public QWidget
     void keyPressEvent(QKeyEvent *event) override;
 
 private:
+    enum class ShowAddNewTorrentDialog {
+        No,
+        Yes,
+        FromSettings,
```

Comment:
This option seems to be redundant.

Reply:
Still in use - when double-clicking on an item.

https://github.com/a-sum-duma/qBittorrent/blob/47efd4a9e31e236f50a3cb097de8d3fdf412fe15/src/gui/search/searchjobwidget.cpp#L174

Reply:
Ok.
But I would rename this enum and its items:
```c++
enum class AddTorrentOption // or AddTorrentBehavior
{
    Default,
    ShowDialog,
    SkipDialog
}
```

Reply:
Fixed.